### PR TITLE
[Docs] Add 1.5.0 change log

### PR DESCRIPTION
**Pull request by jillnogold**

### CODE REVIEW

File: docs/change-log/index.md [hunk 1]
```diff
@@ -1,14 +1,94 @@
...
+
+### Infrastructure
+| ID    | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-3370 | Accessing the MLRun hub is now through a service API. This will enable implementing better function version selection and combining hub functions from different sources. Tech-preview. [View in Git](https://github.com/mlrun/mlrun/pull/3384).|
+| ML-3644 | Supports self-signed docker registries. See [Using self-signed registry](../runtimes/image-build.html#using-self-signed-registry) and [view in Git](https://github.com/mlrun/mlrun/pull/4013). |
+| ML-4132 | New remote function `http_client_kwargs` used to pass any parameter supported in the requests.request method. See [invoke](../api/mlrun.runtimes.html#mlrun.runtimes.RemoteRuntime.invoke) and [view in Git](https://github.com/mlrun/mlrun/pull/3872). |
+| ML-4366 | The MLRun images `mlrun/ml-models` and `mlrun/ml-models-gpu` were deprecated. The new model mlrun/mlrun-gpu is added. Additional dependencies must be installed on an as-need basis. See [MLRun images](../runtimes/images.html#mlrun-images). |
+
```
Comment (liranbg) [inline docs/change-log/index.md:47]:
```suggestion
| ML-4366 | The MLRun images `mlrun/ml-models` and `mlrun/ml-models-gpu` were deprecated and removed. The new model mlrun/mlrun-gpu is added. Additional dependencies must be installed on an as-need basis. See [MLRun images](../runtimes/images.html#mlrun-images). |

```

something to mention they are not available anymore

File: docs/runtimes/configuring-job-resources.md [hunk 2]
```diff
@@ -162,7 +162,8 @@ it would have to be designed so that the job/function state will be saved when s
 
 Preemption mode has three values:
 - Allow: The function pod can run on a spot node if one is available.
-- Constrain: The function pod only runs on spot nodes, and does not run if none is available.
+- Constrain: The function pod only runs on spot nodes, and does not run if none is available. (To set a cluster with constrain 
+   mode, [contact customer support](mailto:support@iguazio.com).)
```
Comment (liranbg) [inline docs/runtimes/configuring-job-resources.md:166]: 👀 why customer support?

Comment (jillnogold) [reply docs/runtimes/configuring-job-resources.md:166]: @liranbg  https://iguazio.slack.com/archives/CN0M28H1S/p1692533912775789?thread_ts=1692532778.029659&cid=CN0M28H1S

File: docs/change-log/index.md [hunk 3]
```diff
@@ -1,14 +1,94 @@
...
+## v1.5.0 (5 October 2023)
+
+### Alerts
+| ID    | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-3854 | Add webhook notification. See webhook in [Notification Kinds](../concepts/notifications.html#notification-kinds) and [view in Git](https://github.com/mlrun/mlrun/pull/3946). |
+
+
+### Data
```
Comment (assaf758) [inline docs/change-log/index.md:22]: Data => Datastore

File: docs/change-log/index.md [hunk 4]
```diff
@@ -646,108 +728,117 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+### APIs that will be removed from v1.6.0 code
+
+| To be removed |Deprecated | API                       | Use instead             |
+| -------| -------| ------------------------------------------------ | --------------------------------------------- |
+| v1.6.0 |  v1.4.0 | MLRunProject.clear_context() | This method deletes all files and clears the context directory or subpath (if defined). This method can produce unexpected outcomes and is not recommended.  |
+| v1.6.0 |  v1.4.0 | MLRunProject object legacy parameters | metadata and spec  |
+| v1.6.0 |  v1.4.0 | BaseRuntime.with_commands and KubejobRuntime.build_config 'verify_base_image' param | 'prepare_image_for_deploy' |
+| v1.6.0 |  v1.4.0 | run_local | function.run(local=True) |
+| v1.6.0 |  v1.4.0 | CSVSource's time_field parameter | Use parse_dates to parse timestamps |
+| v1.6.0 |  v1.4.0 | default_final_state | default_final_step |
```
Comment (assaf758) [inline docs/change-log/index.md:768]: default_final_state  => Feature-set set_targets() default_final_state

**Review by assaf758 — CHANGES_REQUESTED** [review]

File: docs/runtimes/configuring-job-resources.md (hunk 2 above)
Comment (jillnogold) [reply docs/runtimes/configuring-job-resources.md]: @liranbg updated as you suggested

File: docs/change-log/index.md [hunk 5]
```diff
@@ -1,14 +1,95 @@
...
+| --- | ----------------------------------------------------------------- |
+| ML-3501 | Supports running Spark jobs on Databricks cluster. See {ref}`databricks`. [View in Git](https://github.com/mlrun/mlrun/pull/3996). |
+| ML-3854 | Add webhook notification. See webhook in [Notification Kinds](../concepts/notifications.html#notification-kinds) and [view in Git](https://github.com/mlrun/mlrun/pull/3946). |
+| ML-4059 | Supports adding env vars or secrets to the docker build during runtime. See [Extra arguments](../runtimes/image-build.html#extra-arguments),  {py:class}`~mlrun.projects.html#mlrun.projects.MlrunProject.build_config` and [view in Git](https://github.com/mlrun/mlrun/pull/4013). |
+
+
+### UI
+| ID    | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-2811 | New Batch Inference wizard. Tech Preview.|
```
Comment (alonmr) [inline docs/change-log/index.md:47]: maybe move the tech previews to the bottom

File: docs/change-log/index.md [hunk 6]
```diff
@@ -1,14 +1,95 @@
...
+| ML-2815 | The new Batch Run wizard replaces the previous New job. |
+| ML-3584 | The Model Endpoints page now displays the Function Tag. |
+| ML-4066 | The Online types list of the Target Store now includes Redis. | 
+| ML-4167 | Supports downloading the .yaml file from the Projects page. |
+| ML-4571 | The Model Endpoints page now displays the drift threshold and the drift actual value. |
+|ML-4756 | The Recents list in Jobs and Workflows (Projects pane) now displays the last 48 hours. |
```
Comment (alonmr) [inline docs/change-log/index.md:53]:
```suggestion
| ML-4756 | The Recents list in Jobs and Workflows (Projects pane) now displays the last 48 hours. |
```

File: docs/change-log/index.md [hunk 7]
```diff
@@ -1,14 +1,95 @@
...
+
+
+
+### Deprecations
+
+| ID   | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-4075  | Python 3.7 is not supported.|
+| NA | [APIs removed in v1.5.0](##apis-removed-in-v1-5-0-deprecated-in-v1-3-0) and [CLIs removed in v1.5.0](##cli-removed-in-v1-5-0-deprecated-in-v1-3-0).  |
+| ML-4042 | [APIs deprecated in v1.5.0](##apis-deprecated-in-v1-5-0-removed-in-v1-6-0) and [CLIs deprecated in v1.5.0](##cli-deprecated-in-v1-5-0-removed-in-v1-6-0).  |
```
Comment (alonmr) [inline docs/change-log/index.md:77]: I don't see these titles were they added before?
Usually we give 2 minor versions before we remove so the removal should be in 1.7.0 but IIRC there was a special case that we forgot to deprecate and we decided to remove in 1.6.0.

File: docs/change-log/index.md [hunk 8]
```diff
@@ -634,8 +716,10 @@ with a drill-down to view the steps and their details. [Tech Preview]
 | ID   | Description                                            | Workaround                                    | Opened in |
 | ---- | -------------------------------------------------------| --------------------------------------------- | ------ |
 | ML-1584 | Cannot run `code_to_function` when filename contains special characters | Do not use special characters in filenames | v1.0.0 |
+| ML-2030 |  Need means of moving artifacts from test to production Spark | To register artifact between different environments, e.g. dev and prod, upload your artifacts to a remote storage, e.g. S3. You can change the project artifact path using MLRun or MLRun UI. `project.artifact_path='s3:<bucket-name/..'` | 1.0.0 |
```
Comment (alonmr) [inline docs/change-log/index.md:719]:
```suggestion
| ML-2030 |  Need means of moving artifacts from test to production Spark | To register artifact between different environments, e.g. dev and prod, upload your artifacts to a remote storage, e.g. S3. You can change the project artifact path using MLRun or MLRun UI. `project.artifact_path='s3:<bucket-name/..'` | v1.0.0 |
```

File: docs/change-log/index.md [hunk 9]
```diff
@@ -645,109 +729,120 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+### APIs that will be removed from v1.6.0 code
+
+| To be removed |Deprecated | API                       | Use instead             |
+| -------| -------| ------------------------------------------------ | --------------------------------------------- |
+| v1.6.0 |  v1.4.0 | MLRunProject.clear_context() | This method deletes all files and clears the context directory or subpath (if defined). This method can produce unexpected outcomes and is not recommended.  |
+| v1.6.0 |  v1.4.0 | MLRunProject object legacy parameters | metadata and spec  |
+| v1.6.0 |  v1.4.0 | BaseRuntime.with_commands and KubejobRuntime.build_config 'verify_base_image' param | 'prepare_image_for_deploy' |
+| v1.6.0 |  v1.4.0 | run_local | function.run(local=True) |
+| v1.6.0 |  v1.4.0 | CSVSource's time_field parameter | Use parse_dates to parse timestamps |
+| v1.6.0 |  v1.4.0 |  Feature-set set_targets() default_final_state | default_final_step |
```
Comment (alonmr) [inline docs/change-log/index.md:772]:
```suggestion
| v1.6.0 |  v1.4.0 | Feature-set set_targets() default_final_state | default_final_step |
```

File: docs/concepts/workflow-overview.md [hunk 10]
```diff
@@ -149,7 +149,8 @@ completion and prints out the workflow progress. Alternatively, you can use `.wa
 The default workflow engine is `kfp`. You can override it by specifying the `engine` in the `run()` or `set_workflow()` methods. 
 Using the `local` engine executes the workflow state machine locally (its functions still run as cluster jobs).
 If you set the `local` flag to True, the workflow uses the `local` engine AND the functions run as local process.
-This mode is used for local debugging of workflows.
+This mode is used for local debugging of workflows. The `remote` engine runs the workflow from remote. From the project
```
Comment (alonmr) [inline docs/concepts/workflow-overview.md:152]:
```suggestion
This mode is used for local debugging of workflows. The `remote` engine runs the workflow from a remote pod. From the project 
```

**Review by alonmr — COMMENTED** [review]
Minor comments

File: docs/change-log/index.md (hunk 7 above)
Comment (jillnogold) [reply docs/change-log/index.md]: @alonmr That is what I remember also

File: docs/change-log/index.md (hunk 5 above)
Comment (jillnogold) [reply docs/change-log/index.md]: They are in numerical order

File: docs/change-log/index.md (hunk 7 above)
Comment (jillnogold) [reply docs/change-log/index.md]: @alonmr I changed the layout of the deprecations, and now fixed the links.
See https://github.com/mlrun/mlrun/pull/4247

**Pull request merged**